addons/de/xrechnung/instructions.go

```diff
@@ -0,0 +1,107 @@
+package xrechnung
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/pay"
+	"github.com/invopop/validation"
+)
+
+// Payment keys for XRechnung SEPA direct debit and credit transfer
+const (
+	KeyPaymentMeansSEPACreditTransfer cbc.Key = "sepa-credit-transfer"
+	KeyPaymentMeansSEPADirectDebit    cbc.Key = "sepa-direct-debit"
+)
+
+var validPaymentKeys = []cbc.Key{
+	pay.MeansKeyCash,
+	pay.MeansKeyCheque,
+	pay.MeansKeyCreditTransfer,
+	pay.MeansKeyCard,
+	pay.MeansKeyDirectDebit,
+	pay.MeansKeyOther,
+	KeyPaymentMeansSEPACreditTransfer,
+	KeyPaymentMeansSEPADirectDebit,
+}
```

Comment:
Please check the other regime and addon implementations. It is important that the base GOBL types are maintained and only extended if needed with the `+` stuff.